c10/util/complex_type.h

```diff
@@ -208,8 +208,13 @@ struct alignas(sizeof(T) * 2) complex {
     return *this;
   }
 
+#ifdef __APPLE__
+#define FORCE_INLINE_APPLE __attribute__((always_inline))
```

Comment:
@ezyang This is the solution to https://github.com/pytorch/pytorch/issues/39123. I enable this only on MacOS instead of globally, because MSVC seems does not support his.